packaging/upload-itch.sh

```diff
@@ -29,10 +29,11 @@ chmod +x butler
 
 ./butler -V
 ./butler login
-./butler push "${BUILD_OUTPUT_DIR}/OpenRA-${GIT_TAG}-x64.exe" "openra/openra:win" --userversion-file ../VERSION
+zip -u ${BUILD_OUTPUT_DIR}/OpenRA-${GIT_TAG}-x64-winportable.zip .itch.toml
```

Comment:
This is still modifying the file that is pushed to the github release. You need to make a copy somewhere outside ${BUILD_OUTPUT_DIR} first.

Reply:
Did just that.